### PR TITLE
Add User Account dropin docs

### PR DESCRIPTION
**Pull request by keharper**
This PR adds user account dropin docs.

### CODE REVIEW

File: src/content/docs/dropins/user-account/containers/addresses.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+<Diagram caption="Empty minified Addresses container">
+  ![Empty minified Addresses container](@images/dropins/user-account/addresses-minified-multiple.png)
+</Diagram>
+
+<Diagram caption="Minified Addresses container with multiple addresses">
+  ![Minified Addresses container with multiple addresses](@images/dropins/user-account/addresses-minified-multiple.png)
+</Diagram>
+
+The **Selectable**: view is intended for scenarios that require address selection, such as during checkout. This view includes styling adjustments and represents each address card as a selectable radio button, facilitating address selection.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/containers/addresses.mdx]:
```suggestion
The **Selectable** view is intended for scenarios that require address selection, such as during checkout. This view includes styling adjustments and represents each address card as a selectable radio button, facilitating address selection.
```

File: src/content/docs/dropins/user-account/containers/addresses.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        ['formName', 'string', 'No', 'Sets the "name" attribute for the form. Defaults to "addressesForm" if not provided. Useful for custom integrations.'],
+        ['slots.AddressCard', 'slot', 'No', 'Allows overriding address card content, such as rendering additional data or changing layout.'],
+        ['slots.AddressFormActions', 'AddressFormActionsContext', 'No', 'Provides an option to override the call-to-action buttons for the address form.'],
+        ['slots.AddressFormInputs', 'AddressFormInputsContext', 'No', 'Allows additional custom inputs or HTML elements at the form’s bottom.'],
+        ['slots\[`AddressFormInput_${attribute.code}`]', 'slot', 'No', 'Allows overriding input used for a particular form field. Example slot names: “AddressFormInput_firstname”, “AddressFormInput_street”.'],
+        ['title', 'string', 'No', 'Custom container header text overriding translations.'],
+        ['addressFormTitle', 'string', 'No', 'Custom address form header text overriding translations (optional).'],
+        ['defaultSelectAddressId', 'string', 'No', 'Used only in selectable view. Deternines which address is selected by default.'],
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/containers/addresses.mdx]:
```suggestion
        ['defaultSelectAddressId', 'string', 'No', 'Used only in selectable view. Determines which address is selected by default.'],
```

File: src/content/docs/dropins/user-account/containers/addresses.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        ['showSaveCheckBox', 'boolean', 'No', 'Controls visibility of the "Save in address book" checkbox. Useful for implementations where customers can choose to save the address, with form submission managed externally. Enabling this checkbox hides the action buttons automatically (equivalent to setting "hideActionFormButtons" to "true".)'],
+        ['saveCheckBoxValue', 'boolean', 'No', 'Sets the initial value of the "Save in address book" checkbox.'],
+        ['selectable', 'boolean', 'No', 'Whether to enable/disable the selectable view.'],
+        ['className', 'string', 'No', 'Allows custom CSS classes to be applied to the address container.'],
+        ['withHeader', 'boolean', 'No', 'Whether to show/hide the container header.'],
+        ['minifiedView', 'boolean', 'No', 'Whether to enable/disable the minified view.'],
+        ['withActionsInMinifiedView', 'boolean', 'No', 'Whether to show/hide address card actions in the minified view.'],
+        ['withActionsInFullSizeView', 'boolean', 'No', 'Whether to show/hide address card actions in the full size view.'],
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/containers/addresses.mdx]:
```suggestion
        ['withActionsInFullSizeView', 'boolean', 'No', 'Whether to show/hide address card actions in the full-size view.'],
```

File: src/content/docs/dropins/user-account/containers/addresses.mdx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+The `Addresses` container is a standalone component that creates, edits, and deletes customer addresses. It supports custom address attributes and configurable validation through the Adobe Commerce Admin.
+
+The `Addresses` container relies on the [`AddressForm`](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/user-account/containers/address-form/) container for address creation and editing. It inherits all behaviors from the `AddressForm` container.
+
+The Addresses container can be implemented in several views.
+
+The **full-size** view is used when the `Addresses` component functions as a standalone page. This view displays all addresses, enables pagination (based on backend configurations), and provides full functionality for creating, editing, and deleting addresses. A **Create Address** button can be displayed below the list. Clicking the button opens the address creation form, which is configurable from the Commerce Admin.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/containers/addresses.mdx]: Consider adding headings for each view.

File: src/content/docs/dropins/user-account/functions.mdx [hunk 5]
```diff
@@ -5,6 +5,8 @@ description: Learn about the API functions provided by the User Account dropin.
 
 import OptionsTable from '@components/OptionsTable.astro';
 
+The User Account drop-in provides API functions that allow developers to retrieve and display account details, including shipping and billing addresses and previous orders.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/functions.mdx]:
```suggestion
The user account drop-in component provides API functions that allow developers to retrieve and display account details, including shipping and billing addresses and previous orders.
```

File: src/content/docs/dropins/user-account/functions.mdx
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/functions.mdx:1]: Add the following metadata to the file to control the on page nav similar to what we're doing in the [checkout function](https://github.com/commerce-docs/microsite-commerce-storefront/blob/USF-1323-checkout-dropin/src/content/docs/dropins/checkout/checkout-functions.mdx?plain=1#L4C1-L6C21) docs:

```
tableOfContents:
  minHeadingLevel: 2
  maxHeadingLevel: 2
```

File: src/content/docs/dropins/user-account/initialization.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
+---
+title: User Account initialization
+description: Learn how to configure initializer for the User Account drop-in.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/initialization.mdx]:
```suggestion
description: Learn how to configure initializer for the user account drop-in component.
```

File: src/content/docs/dropins/user-account/initialization.mdx [hunk 7]
```diff
@@ -0,0 +1,66 @@
+---
+title: User Account initialization
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/initialization.mdx]:
```suggestion
title: User account initialization
```

File: src/content/docs/dropins/user-account/initialization.mdx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+description: Learn how to configure initializer for the User Account drop-in.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+The User Account drop-in does not require any special data or specific parameters to be initialized. To initialize the drop-in, you must import [`/scripts/initializers/account.js`](https://github.com/hlxsites/aem-boilerplate-commerce/blob/develop/scripts/initializers/account.js) into the appropriate file.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/initialization.mdx]:
```suggestion
The user account drop-in component does not require any special data or specific parameters to be initialized. To initialize the drop-in, you must import [`/scripts/initializers/account.js`](https://github.com/hlxsites/aem-boilerplate-commerce/blob/develop/scripts/initializers/account.js) into the appropriate file.
```

File: src/content/docs/dropins/user-account/initialization.mdx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    default: {
+      ...labels,
+    },
+  };
+
+  return initializers.mountImmediately(initialize, { langDefinitions });
+})();
+```
+
+The `langDefinitions` property defines and registers language packs for the drop-in. This allows you to provide localized text for different languages in your application.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/initialization.mdx]:
```suggestion
The `langDefinitions` property defines and registers language packs for the user account component. This allows you to provide localized text for different languages in your application.
```

File: src/content/docs/dropins/user-account/initialization.mdx [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+## Configuration options
+
+The User Account drop-in initializer accepts the following property:
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/initialization.mdx]:
```suggestion
The user account component initializer accepts the following property:
```

File: src/content/docs/dropins/user-account/sidebar.mdx [hunk 11]
```diff
@@ -0,0 +1,37 @@
+---
+title: User Account Sidebar
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/sidebar.mdx]:
```suggestion
title: User account sidebar
```

File: src/content/docs/dropins/user-account/sidebar.mdx [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+description: Learn about the slots provided in the User Account drop-in.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/sidebar.mdx]:
```suggestion
description: Learn about the slots provided in the user account drop-in component.
```

File: src/content/docs/dropins/user-account/sidebar.mdx [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+The User Account Sidebar is not part of the account drop-in component set, but it is implemented as a block within the boilerplate (`commerce-account-sidebar`). Its primary function is to render a configurable sidebar menu for the user account, managed through SharePoint.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/sidebar.mdx]:
```suggestion
The user account sidebar is not part of the account component, but it is implemented as a block within the boilerplate (`commerce-account-sidebar`). Its primary function is to render a configurable sidebar menu for the user account, managed through SharePoint.
```

File: src/content/docs/dropins/user-account/slots.mdx [hunk 14]
```diff
@@ -0,0 +1,29 @@
+---
+title: User Account Slots
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/slots.mdx]:
```suggestion
title: User account slots
```

File: src/content/docs/dropins/user-account/slots.mdx [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+description: Learn about the slots provided in the User Account drop-in.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/slots.mdx]:
```suggestion
description: Learn about the slots provided in the user account drop-in component.
```

File: src/content/docs/dropins/user-account/slots.mdx [hunk 16]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+Learn about the slots provided in the User Account drop-in.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/slots.mdx]:
```suggestion
Learn about the slots provided in the user account drop-in component.
```

File: src/content/docs/dropins/user-account/slots.mdx [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+
+[Extending drop-ins](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/all/extending/) describes default properties available to all slots.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/slots.mdx]:
```suggestion
[Extending drop-ins components](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/all/extending/) describes default properties available to all slots.
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 18]
```diff
@@ -0,0 +1,114 @@
+---
+title: User Account styles
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
title: User account styles
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 19]
```diff
@@ -0,0 +1,114 @@
...
+description: Learn how to override the styles used within the User Account drop-in.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
description: Learn how to override the styles used within the user account drop-in component.
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 20]
```diff
@@ -0,0 +1,114 @@
...
+import editCustomerInformation from './files/EditCustomerInformation.css?raw';
+import emptyList from './files/EmptyList.css?raw';
+import form from './files/Form.css?raw';
+import ordersListAction from './files/OrdersListAction.css?raw';
+import ordersListCard from './files/OrdersListCard.css?raw';
+import ordersListWrapper from './files/OrdersListWrapper.css?raw';
+import Diagram from '@components/Diagram.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+This topic introduces you to the CSS classes for each UI component used in the User Account drop-in and shows you how to override these classes to customize the drop-in's CSS styling to match your brand.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
This topic introduces you to the CSS classes for each UI component used in the user account drop-in component and shows you how to override these classes to customize the user account component's CSS styling to match your brand.
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 21]
```diff
@@ -0,0 +1,114 @@
...
+
+:::tip[Boilerplate CSS Tip!]
+This drop-in contains multiple components that provide its UI. If you plan to make several CSS changes to the drop-in, we suggest creating CSS files for each drop-in component (in the `blocks/commerce-account/` directory) and importing those into the `commerce-account.css` file as shown below. This practice makes it easier to maintain and update your CSS in the future.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
User account contains multiple components that provide its UI. If you plan to make several CSS changes, Adobe suggests creating CSS files for each drop-in component (in the `blocks/commerce-account/` directory) and importing those into the `commerce-account.css` file as shown below. This practice makes it easier to maintain and update your CSS in the future.
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 22]
```diff
@@ -0,0 +1,114 @@
...
+@import 'orders-list-action.css';
+@import 'orders-list-card.css';
+@import 'orders-list-wrapper.css';
+```
+
+:::
+
+## Big Picture
+
+The quickest way to override User Account CSS is to inspect the User Account UI from your browser's developer tools to discover the BEM class names you want to add to or override. This process is numbered in the image below.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
The quickest way to override user account CSS is to inspect the user account UI from your browser's developer tools to discover the BEM class names you want to add to or override. This process is numbered in the image below.
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 23]
```diff
@@ -0,0 +1,114 @@
...
+1. **Identify the CSS class(es)** for the element. We use [BEM naming](https://getbem.com/naming/), which makes it easy to know which component you're changing (and which CSS file to use).
+1. **Copy the CSS class names to override** to your custom CSS file.
+
+</Callouts>
+
+## How to override the default styles
+
+<Steps>
+
+1. **Create a new CSS file** in your project for the User Account drop-in. Name the file `custom-account.css`.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
1. **Create a new CSS file** in your project for the user account component. Name the file `custom-account.css`.
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 24]
```diff
@@ -0,0 +1,114 @@
...
+   @import 'custom-account.css';
+   ```
+
+1. **Save the file** and refresh your browser to see the changes.
+
+</Steps>
+
+## Example CSS overrides
+
+The following example shows how to override two of the many classes for the User Account drop-in component.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
The following example shows how to override two of the many classes for the user account component.
```

File: src/content/docs/dropins/user-account/styles.mdx [hunk 25]
```diff
@@ -0,0 +1,114 @@
...
+
+.checkout__heading-title {
+  line-height: 1.5;
+  font-size: 1.5rem;
+}
+```
+
+## Account CSS classes
+
+The CSS classes for each User Account drop-in component are provided here.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/styles.mdx]:
```suggestion
The CSS classes for each user account component are provided here.
```

File: src/content/docs/dropins/user-account/tutorials.mdx [hunk 26]
```diff
@@ -0,0 +1,86 @@
+---
+title: User Account Tutorials
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/tutorials.mdx]:
```suggestion
title: User account tutorials
```

File: src/content/docs/dropins/user-account/tutorials.mdx [hunk 27]
```diff
@@ -0,0 +1,86 @@
...
+description: Learn how to implement and customize the User Account drop-in.
```
Comment (jeff-matthews) [inline src/content/docs/dropins/user-account/tutorials.mdx]:
```suggestion
description: Learn how to implement and customize the user account drop-in component.
```

**Review by jeff-matthews — COMMENTED** [review]
Looks good @keharper.

I only made superficial editorial suggestions. Specifically where the approved naming convention applies to "dropins".

Approved convention from internal wiki:

>- "Drop-in components" is preferred over "DropIns."
the name structure [Functional Name] Drop-in is correct for individual drop-ins. Acronyms should not be used as part of the name. (from Branding)
>
>- "Cart drop-in component", once established, can be written as, "the cart component" or "the cart").

File: src/content/docs/dropins/user-account/containers/address-form.mdx [hunk 28]
```diff
@@ -0,0 +1,119 @@
...
+
+```javascript
+const formRef = { current: null };
+
+provider.render(AddressForm, {
+  forwardFormRef: formRef,
+})(containerWrapper);
+
+const isFormValid = formRef.current.handleValidationSubmit();
+const formData = inputRef.current.formData;
```
Comment (KostiantynFandeliuk) [inline src/content/docs/dropins/user-account/containers/address-form.mdx:72]: return {
          handleValidationSubmit,
          formData: normalizeGetAddressData(modifyFormRef, true),
          isDataValid,
        };
        
        isDataValid - A new boolean value has been added. It indicates whether the form has been validated or not without an explicit submit action.

**Review by KostiantynFandeliuk — CHANGES_REQUESTED** [review]

**Review by KostiantynFandeliuk — APPROVED** [review]
Looks good to me. Thank you.

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**